tests/query/highlights/swift/string-interpolation.swift

```diff
@@ -0,0 +1,7 @@
+var one = 1
+var two = 2
+var name = "Let's do some math: \(one) + \(one) = \(one + one) = \(two)"
+//                              ^ <- punctuaction.bracket
+//                                 ^ <- variable
+//                                   ^ <- punctuaction.bracket
+//                                                 ^ <- punctuaction.bracket
```

Comment:
`CUSTOM_OPERATOR` as using in `for x in 0..<3` can not be highlighted at the moment

Reply:
`..<` can in some cases.